### PR TITLE
Clean-up of files following 2019.05.16

### PR DESCRIPTION
**Pull request by lcoombe**

### CODE REVIEW

File: orca-0/Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ less \
 libxml2 \
 matplotlib \
 miller \
+mysql \
```
Comment (sjackman) [inline orca-0/Dockerfile]: I don't think we need `mysql` in the image if nothing is using it.

File: Dockerfile [hunk 2]
```diff
@@ -5,21 +5,3 @@ LABEL maintainer="Shaun Jackman <sjackman@gmail.com>" \
       org.label-schema.url="http://www.bcgsc.ca/services/orca" \
       org.label-schema.vcs-url="https://github.com/bcgsc/orca" \
       org.label-schema.vendor="BC Cancer Genome Sciences Centre"
-
-RUN sudo apt-get update \
-    && sudo apt-get install -y --no-install-recommends man-db \
-    && sudo rm -rf /var/lib/apt/lists/*
```
Comment (sjackman) [inline Dockerfile]: Can you please add `man-db` back to `orca-0/Dockerfile`? I forget why we removed it though, and the PR doesn't say. https://github.com/bcgsc/orca/pull/82

Comment (sjackman) [reply Dockerfile]: other than it failed to install. I think it conflicted with another package.

Comment (lcoombe) [reply Dockerfile]: I did go back and add `man-db` to `orca-0/Dockerfile` in a previous PR - it should be there

File: orca-6/Dockerfile [hunk 3]
```diff
@@ -117,5 +112,4 @@ wiggletools \
 wtdbg2 \
 xmatchview \
 xssp \
-yaha \
-vim
```
Comment (sjackman) [inline orca-6/Dockerfile]: Can you please add `vim` back to orca-0/Dockerfile?

Comment (lcoombe) [reply orca-6/Dockerfile]: `vim` is there already :) This temporary addition was a hack because I had to uninstall ruby

File: Dockerfile (hunk 2 above)
Comment (sjackman) [reply Dockerfile]: It doesn't look like `man-db` is there, but `vim` is there.

File: orca-6/Dockerfile (hunk 3 above)
Comment (sjackman) [reply orca-6/Dockerfile]: Never mind. `vim` is already in there.

File: Dockerfile (hunk 2 above)
Comment (lcoombe) [reply Dockerfile]: Here, right? ```# fonts-dejavu-core is required by graphviz.
RUN sudo apt-get update \
        && sudo apt-get install -y --no-install-recommends \
                fonts-dejavu-core man-db \
        && sudo rm -rf /var/lib/apt/lists/*
```

File: orca-6/Dockerfile (hunk 3 above)
Comment (sjackman) [reply orca-6/Dockerfile]: Jinx! Thanks, Lauren.

File: Dockerfile (hunk 2 above)
Comment (lcoombe) [reply Dockerfile]: I believe the `brew install man-db` wasn't working, so we added it in this way

Comment (sjackman) [reply Dockerfile]: Ah, I see. You can remove it from `apt-get` and add it to `brew install`.

File: orca-0/Dockerfile (hunk 1 above)
Comment (lcoombe) [reply orca-0/Dockerfile]: Done!

File: Dockerfile (hunk 2 above)
Comment (lcoombe) [reply Dockerfile]: Sounds good -- done!

Comment (sjackman) [reply Dockerfile]: Thanks!

**Pull request merged**